build/Common.prod.props

```diff
@@ -10,6 +10,16 @@
       <PrivateAssets>All</PrivateAssets>
     </PackageReference>
   </ItemGroup>
+  <PropertyGroup>
+    <AdditionalApiCompatOptions>--exclude-non-browsable --exclude-compiler-generated</AdditionalApiCompatOptions>
```

Comment:
What do these options do?

Reply:
These options are not actually needed at the moment, I'll remove them.. they're defined [here](https://github.com/dotnet/arcade/blob/main/src/Microsoft.DotNet.ApiCompat/src/ApiCompatRunner.cs#L76)